systray.go

```diff
@@ -20,6 +20,11 @@ var (
 	quitOnce  sync.Once
 )
 
+var (
```

Comment:
I don't think we need another var block.
And is this comment should probably be removed - it's not required and it's out of date now.